CMakeLists.txt

```diff
@@ -333,15 +354,12 @@ endif()
 
 option(WITH_APPS "Build apps" ON)
 if (WITH_APPS)
-  if (NOT WIN32)
-    message(STATUS "Building apps enabled")
-    add_subdirectory(apps)
-  else()
-    message(WARNING "Apps cannot be built under Windows using cmake. Try the makefile in an msys2 shell.")
-  endif()
+  message(STATUS "Building apps enabled")
```

Comment:
Thank you so much for doing this! It's been on my neglected list for way too long.

Reply:
NP!